src/models/db/conditionFulfillment.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Model = require('five-bells-shared').Model
+const PersistentModelMixin = require('five-bells-shared').PersistentModelMixin
+const JsonField = require('sequelize-json')
+const validator = require('../../services/validator')
+const Sequelize = require('sequelize')
+const sequelize = require('../../services/db')
+const Transfer = require('./transfer').Transfer
+const parseBody = require('co-body')
+const InvalidBodyError = require('five-bells-shared/errors/invalid-body-error')
+
+class ConditionFulfillment extends Model {
+
+  // @override
+  static createBodyParser () {
+    const Self = this
+
+    return function * (next) {
+      let json = yield parseBody(this)
+      const validationResult = Self.validateExternal(json)
+      if (validationResult.valid !== true) {
+        const message = validationResult.schema
+          ? 'Body did not match schema ' + validationResult.schema
+          : 'Body did not pass validation'
+        throw new InvalidBodyError(message, validationResult.errors)
+      }
+
+      const model = new Self()
+      model.setConditionFulfillment(json)
+      model.setTransferId(this.params.id)
```

Comment:
Yeah, I actually wasn't sure the best way to handle this. We need to validate the payload against the fulfillment schema, but then we need to attach the `transfer_id` (HTTP param) to the payload, since that is how it's stored in the db. What do you think the best way to do this is?